docs/compose/FloatingActionButton/README.md

```diff
@@ -0,0 +1,21 @@
+# Floating Action Button
+
+## Installation
+
+Backpack Compose is available through [Maven Central](https://search.maven.org/artifact/net.skyscanner.backpack/backpack-compose). Check the main [Readme](https://github.com/skyscanner/backpack-android#installation) for a complete installation guide.
+
+## Usage
+
+Example of a `BpkFab`:
+
+```Kotlin
+import net.skyscanner.backpack.compose.button.BpkFab
```

Comment:
This import is incorrect. Also, there're imports missing for BpkIcon and Search – you can find those in your story.